nncf/quantization/algorithms/min_max/backend.py

```diff
@@ -153,6 +146,16 @@ def create_quantizer_insertion_command(
         :return: Backend-specific TransformationCommand for the quantizer insertion operation.
         """
 
+    @staticmethod
+    @abstractmethod
+    def get_start_nodes_for_activation_path_tracing(nncf_graph: NNCFGraph) -> List[OperatorMetatype]:
```

Comment:
The method is called _get_start__**nodes**__for_activation_path_tracing_, but returns the list of the `OperatorMetatype`.

Reply:
Done